### DISCUSSION TITLE
Some ideas from a syntheticism lover

### DISCUSSION THREAD

**Discussion by JoaoVitorSCS1999** [FakeFishGames/Barotrauma#7426] [Ideas and suggestions]
- Thalamus Symbiosis
- Cloning mechanics, cloning someone of your crew by getting the member in question stats and with a chance o failure generating a new variation of human with resetted stats of that human cloned (with GFX idk like coraline spines or melting faces) also feral failed clones
- Player homes/outposts/vaults
- androids, robots that the player could program to make certain actions, parts so you could customize that robot, each part with different functions
- GFX for the genes mechanics attributes making human have other species characteristics matching the respective buff
- Monster taming
- Monster Breeding
- Body types
- Hunger/thirst/sleep mechanics
- Fractal surgery so you put a dead crew member brain into a fractal guardian body making it like some sort of organic drone
- Room assignment

**Comment by MaaiG** [comment]
Body types - Yeees!!!

**Comment by ghost** [comment]
Barnacles would be fitting too

**Comment by WickedWolfy** [comment]
I would be against `Hunger/thirst/sleep mechanics` being a core mechanics. Maybe an option to enable/disable for the game.
